data-model/src/main/java/org/alfresco/repo/content/ContentStore.java

```diff
@@ -344,4 +345,38 @@ default DirectAccessUrl requestContentDirectUrl(String contentUrl, boolean attac
     {
         return Collections.emptyMap();
     }
+
+    /**
+     * Submit a request to send content to archive (offline) state.
+     * If no connector is present or connector is not supporting sending to archive, then {@code false} will be returned.
```

Comment:
Comment says if no connector is present it will return false (but default throws an exception) ?

Reply:
Rephrased javadoc to fit into real behaviour.